src/include/squid/complex.rb

```diff
@@ -49,11 +49,98 @@ def initialize_squid_complex(include_target)
       Yast.include include_target, "squid/store_del.rb"
       Yast.include include_target, "squid/inits.rb"
 
-
       @main_caption = _("Squid")
+      @widget_descr = {}
+      @screens = {}
+
+      load_widgets
+      load_screens
+    end
+
+    def ReallyAbort
+      !Squid.GetModified || Popup.ReallyAbort(true)
+    end
+
+    def PollAbort
+      UI.PollInput == :abort
+    end
+
+    # Read settings dialog
+    #
+    # @return [Symbol] :abort when settings could not be read; :next otherwise
+    def ReadDialog
+      Wizard.RestoreHelp(Ops.get_string(@HELPS, "read", ""))
```

Comment:
```suggestion
      Wizard.RestoreHelp(@HELPS["read"]))
```

Reply:
You're right.

This time I forgot to update old code, focusing only in fix the problem.

Thanks.

Reply:
I took the opportunity to test the "Apply suggestion" button :)